config/proposer/loader/loader.go

```diff
@@ -128,12 +129,22 @@ func (psl *settingsLoader) Load(cliCtx *cli.Context) (*proposer.Settings, error)
 			return nil, err
 		}
 		loadConfig = dbps.ToConsensus()
+		log.Debugf("DB Loaded Proposer Settings: %s", func() string {
+			b, err := json.Marshal(loadConfig)
+			if err != nil {
+				return err.Error()
+			}
+			return string(b)
+		}())
 	}
 
 	// start to process based on load method
 	for _, method := range psl.loadMethods {
 		switch method {
 		case defaultFlag:
+			if psl.existsInDB {
+				log.Warn("Previously saved proposer settings were loaded from the DB, only default settings will be updated. Please provide new proposer settings or clear DB to reset proposer settings.")
```

Comment:
If I start the VC with a fresh DB:

```
bazel run //cmd/validator -- --holesky --suggested-fee-recipient 0x3A8734b7D0a66b0a99218a8dAd27933e50A65efB --validators-external-signer-url http://localhost:9000 --proposer-settings-file  /Users/manu/Downloads/proposer_settings.yaml
```

==> OK

If I restart the VC with the same command:
```
[2024-06-13 11:39:02]  WARN Previously saved proposer settings were loaded from the DB, only default settings will be updated. Please provide new proposer settings or clear DB to reset proposer settings.
```

By `default settings` are you talking about `defaultconfig` (by opposition to `proposerconfig`?
If yes, why do only the `default settings` will be updated in such a case?

Reply:
hmm you're right it doesn't i should probably change this log

Reply:
if you provide the proposer settings file flag it should also be updated.